source/state_representation/include/state_representation/trajectory/CartesianTrajectory.hpp

```diff
@@ -0,0 +1,165 @@
+#pragma once
+
+#include "state_representation/space/cartesian/CartesianState.hpp"
+#include "state_representation/trajectory/TrajectoryBase.hpp"
+
+namespace state_representation {
+class CartesianTrajectory : public TrajectoryBase<Eigen::VectorXd> {
```

Comment:
```suggestion
class CartesianTrajectory : public TrajectoryBase<Eigen::Vector<double, 25>> {
```

Reply:
here for us to set the size at compile time we would need to use something like

```cpp
using Vector25d = Eigen::Matrix<double, 25, 1>;
```

At which point the function parameters would either need to be of that type, or we would need to have some special handling of `Eigen::VectorXd` parameters (direct assignments will cause exceptions because on is fixed and the other dynamic)